cirq-google/cirq_google/study/finite_random_variable.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2025 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Any, Iterator
+
+import cirq
+from cirq.study.sweeps import SingleSweep
+
+
+class FiniteRandomVariable(SingleSweep):
+    """A sweep over randomly-sampled values from a finite distribution.
+
+    This can generate a stream of random samples pulled from a finite
+    distribution.  For instance, a coin flip, die roll, or from {1, 0, -1}.
```

Comment:
Grammar nit: the "For instance …" part is not a complete sentence.

Reply:
Done.